### ISSUE TITLE
Submit button not working

### ISSUE THREAD

**Issue by nscpro** [THE-WIRE/web-rt#3]
Hi, 

I fill all the fields and press "Send Invitation Mails" button, but nothing change / happened. Same to Join Team buttom.

**Comment by itssuyog96** [comment]
Hi @nscpro,

You have to configure your mail client for that.

Actually we have not set a notification service for that so it will not show anything. But its working fine unless you make an invalid entry. So, if you are sure enough that you have made a valid entry then just go to your Visual Studio Code Editor and enter your credentials to start realtime collaboration.

Btw what OS are you using?

**Comment by nscpro** [comment]
I'm using MacOS Sierra

**Comment by nscpro** [comment]
Sorry, but i don't understand how does this work, when i go localhost:4200 site and click "Create a new team" button. I fill all field.

Username: mario
Email: mario@nsc.ee
Password: nsc123
Team name: nscpro 

and then i click "Send Invitation Mails" button. Next i go Visual Studio Code, what can i do next? 

File -> Open -> RT-Code-Collab (folder) open and next i press CMD + Shift + P and type "start wire" .

I write email mario@nsc.ee and password nsc123, but nothing work. What can i do wrong?

**Comment by itssuyog96** [comment]
Is there any error you are getting?
Bcoz the process you followed is correct.

Did your team mate got all the files you were sharing?

**Comment by nscpro** [comment]
Yes, Visual Studio Code show me "The password is invalid or the user does not have a password." And thats it.
